src/util/timezones.cpp

```diff
@@ -39,6 +57,13 @@ Timezoner::Timezoner(const char geojson[], std::time_t utc_time_now)
                                     std::to_string(code) + " malformed at offset " +
                                     std::to_string(offset));
     }
+
+    // Run-time check for time zones data: 1500000000 ↔ 'Thu Jul 13 22:40:00 EDT 2017'
+    if (getLocalTimeinfo("EST+5EDT", 1500000000).tm_hour != 22)
```

Comment:
@karenzshea Rethinking about the check: better to move it out of OSRM codebase and document only  `tzdata` requirement and making check as
`[ $(TZ=EST+5EDT date -d @1500000000 +%H) == 22 ]  || echo "tzdata is not installed"`
once before the first time updater run.

Or as a full check
`[ $(TZ=EST+5EDT date -d @1500000000 +%Y%m%d%H%M%S) == 20170713224000 ] || echo "tzdata is not installed"`